### DISCUSSION TITLE
npm install errors

### DISCUSSION THREAD

**Discussion by geniusdev329** [community/community#51068] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I used to get some npm install errors with React over 18.0 version.
Please help me to solve this issue.

**Comment by cargood** [comment, marked as answer]
Solutions:
1. You can use the versions that is less than 18.0 (recommend 17.0)
    When you try with this way, you need to reinstall "react-scripts' small version again. (For example, 4.0)
2. npm install --legacy-peer-deps
